about.html

```diff
@@ -10,56 +10,67 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<header>
+  <section class="Top">
```

Comment:
This section is good, you can leave out the <section>and just use header and nav.  Informally, section is a section to a document and the header/nav is not a section in a document.